docs/Other SDL Features/Creating an App Service/index.md

```diff
@@ -746,3 +746,62 @@ sdlManager.addOnRPCRequestListener(FunctionID.PERFORM_APP_SERVICES_INTERACTION,
 });
 ```
 !@
+
+## Updating Your Published App Service
+Once you have published your app service, you may decide to update the type of data provided in your app service. For example, you may want to give all-access to paid subscribers but provide limited information to free users. You can do this by updating the app service manifest. If desired, you can also delete your app service by unpublishing the service. 
+
+### 7. Updating a Published App Service Manifest (RPC v6.0+)
+@![iOS]
+##### Objective-C
+```objc
+SDLAppServiceManifest *manifest = [[SDLAppServiceManifest alloc] initWithAppServiceType:SDLAppServiceTypeWeather];
+manifest.weatherServiceManifest = <#Updated weather service manifest#>
+
+SDLPublishAppService *publishServiceRequest = [[SDLPublishAppService alloc] initWithAppServiceManifest:manifest];
+[self.sdlManager sendRequest:publishServiceRequest];
+```
+
+##### Swift
+```swift
+let manifest = SDLAppServiceManifest(appServiceType: .weather)
+manifest.weatherServiceManifest = <#Updated weather service manifest#>
+
+let publishServiceRequest = SDLPublishAppService(appServiceManifest: manifest)
+sdlManager.send(publishServiceRequest)
+```
+!@
+
+@![android,javaSE,javaEE]
+##### Java
```

Comment:
```suggestion
```

They don't usually provide this since they only have one language, though I see it in this document so... ¯\_(ツ)_/¯

Probably should remove it throughout the document.